### DISCUSSION TITLE
Problems with name changed

### DISCUSSION THREAD

**Discussion by KatherineJA** [community/community#140830] [GitHub Education] [answered]
### Hi everyone,

Hello! 
I just checked and I have this : Congrats! Your request for [@KathSSs](https://github.com/KathSSs) was approved and your benefits will be available [soon](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits).

But I am really worried about it because I just changed my user name because our Uni told us so, so it would be more "Profesional", so do I have to ask for git hub education again?
I did not use other email or anything I just change my user name
Thank you

**Comment by davevad93** [comment, marked as answer]
Hi @KatherineJA ,

Changing the username shouldn't affect your application as indicated in the following discussions: #52888, #75262

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @KatherineJA was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
